src/views/ConnectWallet.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div>
+    <div class="p-sm-2">
+      <div class="my-3">
+        <div>
+          <h1>connect wallet</h1>
+          <p>Connect your Bitcoin or Lightning wallet to your Umbrel</p>
+        </div>
+      </div>
+
+      <b-row>
+        <b-col cols="12" md="4" xl="3">
+          <b-form-select
+            :value="wallet"
+            :options="options"
+            @change="selectWallet"
+            class="mb-4"
+          ></b-form-select>
+        </b-col>
+      </b-row>
+
+      <router-view :urls="urls" @showQrModal="showQrModal"></router-view>
+    </div>
+
+    <b-modal
+      ref="qr-modal"
+      :title="`${this.qrModalData.wallet} QR Code`"
+      no-close-on-backdrop
+      ok-only
+      size="lg"
+    >
+      <div class="d-flex w-100 align-items-center justify-content-center">
+        <qr-code
+          :value="this.qrModalData.value"
+          :size="500"
+          class="qr-image mt-2"
+          showLogo
+        ></qr-code>
+      </div>
+    </b-modal>
+  </div>
+</template>
+
+<script>
+import { mapState } from "vuex";
+import QrCode from "@/components/Utility/QrCode.vue";
+
+export default {
+  data() {
+    return {
+      options: [
+        { value: null, text: "Select your wallet", disabled: true },
+        {
+          label: "Bitcoin",
```

Comment:
I don't think we need to make the distinction between "Bitcoin" and "Lightning" here. It could be confusing for a few reasons:

- Lightning is Bitcoin
- Many wallets listed here support both on-chain and Lightning Bitcoin
- Many users may not know if they're using Lightning or not

We should just list "Wallets" with all the wallets, and then maybe "Connection Details" with the raw protocols.

Reply:
Sure, seems a lot more logical.

Reply:
Resolved in bae1bd1.